drivers/hmis/app/jobs/hmis/auto_exit_job.rb

```diff
@@ -0,0 +1,80 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Hmis
+  class AutoExitJob < BaseJob
+    def perform
+      Hmis::Hud::Project.all.each do |project|
+        config = Hmis::AutoExitConfig.config_for_project(project)
+        next unless config.present?
+
+        project.enrollments.each do |enrollment|
```

Comment:
We can skip exited enrollments and WIP enrolments
```suggestion
        project.enrollments.open_excluding_wip.each do |enrollment|
```